MatrixSDK/Data/MXRoomSummary.m

```diff
@@ -480,6 +480,30 @@ - (void)unregisterEventEditsListener
 
 #pragma mark - Trust management
 
+- (void)enableTrustTracking:(BOOL)enable
+{
+    if (enable)
+    {
+        if (!_isEncrypted || _trust)
+        {
+            // Not applicable or already enabled
+            return;
+        }
+        
+        NSLog(@"[MXRoomSummary] enableTrustTracking: YES");
+        
+        // Bootstrap trust computation
+        [self registerTrustLevelDidChangeNotifications];
+        [self triggerComputeTrust:YES];
+    }
+    else
+    {
+        NSLog(@"[MXRoomSummary] enableTrustTracking: NO");
+        [self unregisterTrustLevelDidChangeNotifications];
+        _trust = nil;
+    }
+}
+
 - (void)setIsEncrypted:(BOOL)isEncrypted
 {
     _isEncrypted = isEncrypted;
```

Comment:
Should we also consider `_trust` here at line `if (_isEncrypted && [MXSDKOptions sharedInstance].computeE2ERoomSummaryTrust)`, like we did below? Or it's not applicable because the room is newly being encrypted, so we won't have a `_trust` yet?

`if (_isEncrypted && ([MXSDKOptions sharedInstance].computeE2ERoomSummaryTrust || _trust))`

Reply:
This is the latter. `_trust` will be nil.

Reply:
Great, thanks.